### PR TITLE
fix(boards): send mail to new board members

### PR DESCRIPTION
**Pull request by WikiRik**
Still need to test it locally and add tests

### CODE REVIEW

File: test/assets/core-permissions-full.json [hunk 1]
```diff
@@ -55,5 +55,8 @@
     },
     {
       "combined": "global:manage_network:fulfilment_email"
+    },
+    {
+      "combined": "global:see_background_tasks:network"
```
Comment (LeonVreling) [inline test/assets/core-permissions-full.json]: Is this needed?

Comment (WikiRik) [inline test/assets/core-permissions-full.json]:
```suggestion
```

**Review by LeonVreling — CHANGES_REQUESTED** [review]

File: test/assets/core-permissions-full.json (hunk 1 above)
Comment (WikiRik) [reply test/assets/core-permissions-full.json]: I don't see it anymore

**Review by LeonVreling — APPROVED** [review]

**Pull request merged**